pkg/legacyapis/controlplane/register.go

```diff
@@ -58,6 +58,7 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&newcontrolplane.NetworkPolicyStatus{},
 		&newcontrolplane.NodeStatsSummary{},
 		&newcontrolplane.ClusterGroupMembers{},
+		&newcontrolplane.PaginationGetOptions{},
```

Comment:
We don't need to add support for legacy APIs. They will be removed soon.

Reply:
+1. Actually I think we can remove them for v1.6. Let me see if I can open a PR.

Reply:
👌 Sure, though keeping this before the removal PR gets merged to build successfully. Because during API initialization it looks for this type in group version. Tested that w/o this registration, controller would throw an error.

Reply:
legacyAPI has been removed, you could rebase on main now.